lib/components/narrative/default/default-itinerary.js

```diff
@@ -210,14 +210,14 @@ class DefaultItinerary extends NarrativeItinerary {
     return (
       <div
         className={`option default-itin${active ? ' active' : ''}${expanded ? ' expanded' : ''}`}
-        role='presentation'
+        onMouseEnter={this._onMouseEnter}
         // FIXME: Move style to css
```

Comment:
Comment out of place

Reply:
Moved in https://github.com/opentripplanner/otp-react-redux/pull/390/commits/11773ca2755c074d3ca83db0e40a92031c737b97.